03-routing-and-input.Rmd

```diff
@@ -238,6 +238,7 @@ Name | Example | Description
 `SERVER_NAME` | `"127.0.0.1"` | The host portion of the incoming request. You may favor `HTTP_HOST`, if available.
 `SERVER_PORT` | `"8000"` | The target port for the request
 `HTTP_*` | `"HTTP_USER_AGENT"` | Entries for all of the HTTP headers sent with this request.
+`args` | `"a=123"` | A named vector of the `QUERY_STRING`
```

Comment:
I worry that the example here may be misleading since it's presented as a string. Perhaps we could use the syntax we use with `cookies` above. And pardon my rusty memory, but is the args value a proper list? Or is it just a named vector?

Reply:
Was contradicting myself, list indeed.